### ISSUE TITLE
[Bug]: Prompt not being shown in PhpStorm having the new UI preview enabled

### ISSUE THREAD

**Issue by crezra** [laravel-idea/plugin#584]
### Bug description

When using the new UI preview, the plugin would not open a prompt upon selecting "Other" in the generation dropdown menu, yet, all the pinned generations do work and show a modal to create the selected resource.

Preview https://share.cleanshot.com/BTXwcj

This is behavior has only been encountered with the New UI EA Preview for PhpStorm.

Versions : 
`Laravel 9.1` 
`PhpStorm 2022.2 (222.3345.21)`

### Plugin version

5.6.2.222

### Operating system

MacOS

### Steps to reproduce

- ⌘+N to create a new file
- Choose Laravel IDEA menu
- Select "Other"
- No prompt opens

**Comment by adelf** [comment]
Does it appear when you use the main menu > Laravel > Code Generation...(it's basically the same thing)?

**Comment by crezra** [comment]
> Does it appear when you use the main menu > Laravel > Code Generation...(it's basically the same thing)?

Yes, it does appear. So I guess it only happens when selecting "Other" in the file context menu

**Comment by juse-less** [comment]
@crezra Would it be possible for you to test turning off some plugins?
The prompt opens for me with the versions of Laravel Idea and the new JetBrains UI preview shown in the screenshot.
I'm on PhpStorm 2022.2 as well.

<img width="322" alt="ScreenShot 2022-08-10 at 00 47 22" src="https://user-images.githubusercontent.com/76173223/183774779-74cea682-f3a8-477b-8b10-30328ab72eeb.png">

I did a quick screen recording from a basic Laravel + Nova project.
I noticed I haven't changed any settings or set up any 'make' commands, etc.
But it works the same in larger, very custom projects I'm working on.
I don't use many plugins in PhpStorm, and have even turned off many built-in ones.

<details>
<summary>Screen recording</summary>

https://user-images.githubusercontent.com/76173223/183775483-4675a996-739d-41b9-974b-55f0d8f0da8e.mp4
</details>
